jira7/jira_connector_endpoint.bal

```diff
@@ -21,48 +21,45 @@ package jira7;
 import ballerina/io;
 import ballerina/http;
 
-public struct JiraConfiguration {
-    http:ClientEndpointConfiguration httpClientConfig;
-    string url;
-    string username;
-    string password;
-}
-
-public function <JiraConfiguration jiraConfig> JiraConfiguration () {
-    jiraConfig.httpClientConfig = {};
-}
-
-public struct JiraEndpoint {
-    JiraConfiguration jiraConfig;
-    JiraConnector jiraConnector;
-}
-
-public function <JiraEndpoint jiraEP> init (JiraConfiguration userConfig) {
-
-    http:ClientEndpointConfiguration httpConfig = {targets:[{url:userConfig.url + JIRA_REST_API_RESOURCE +
-                                                                 JIRA_REST_API_VERSION}],
-                                                      chunking:http:Chunking.NEVER
-                                                  };
-    userConfig.httpClientConfig = httpConfig;
-
-    jiraEP.jiraConnector = {
-       jiraHttpClientEPConfig:jiraEP.jiraConfig.httpClientConfig,
-       jira_base_url:userConfig.url,
-       jira_authentication_ep:userConfig.url + JIRA_AUTH_RESOURCE,
-       jira_rest_api_ep:userConfig.url + JIRA_REST_API_RESOURCE + JIRA_REST_API_VERSION
-    };
-
-    jiraEP.jiraConnector.setBase64EncodedCredentials(userConfig.username, userConfig.password);
-
-    jiraEP.jiraConfig = {url:userConfig.url, httpClientConfig:userConfig.httpClientConfig};
-    jiraEP.jiraConnector.jiraHttpClientEP.init(jiraEP.jiraConfig.httpClientConfig);
-}
-
-@Description {value:"Returns the connector that client code uses"}
-@Return {value:"The connector that client code uses"}
-public function <JiraEndpoint jiraConnectorEP> getClient () returns JiraConnector {
-    return jiraConnectorEP.jiraConnector;
-}
+public type JiraConfiguration {
+            http:ClientEndpointConfiguration httpClientConfig;
+string url;
+string username;
+string password;
+};
+
+
+
+public type JiraEndpoint object {
```

Comment:
Please rename Endpoint to client.

Reply:
Renamed the endpoint object as "JiraClient"